packages/sdk-demo/src/core/fetchToken.ts

```diff
@@ -24,14 +24,13 @@ export async function fetchToken(
   });
 
   if (!res.ok) {
-    throw new Error('Could not fetch token');
+    throw new Error(`Could not fetch token: ${await res.text()}`);
```

Comment:
It might return empty string from that promise and the sentence could make no sense ending with " : "

```ts
const errorMessage = await res.text();
throw new Error(`Could not fetch token: ${errorMessage || 'Unknown error'}`);
```

Reply:
In our case, it can't be an empty string.